usuario/prisma/schema.prisma

```diff
@@ -11,61 +11,103 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+model Material {
+  id           Int       @id @default(autoincrement())
+  name         String
+  descripcion  String
+  stock        Int
+  costo        Float
+  proveedor_id Int
+  Proveedor    Proveedor @relation(fields: [proveedor_id], references: [id])
+  createdAt    DateTime  @default(now())
+  updatedAt    DateTime  @updatedAt
+
+  @@map("materiales")
+}
+
+model Presupuesto {
+  id           Int       @id @default(autoincrement())
+  descripcion  String
+  recepcion_id Int
+  Recepcion    Recepcion @relation(fields: [recepcion_id], references: [id])
+  client_id    Int
+  Usuario      Usuario   @relation(fields: [client_id], references: [id])
+  estado       String
+  confirmado   Boolean
+  createdAt    DateTime  @default(now())
+  updatedAt    DateTime  @updatedAt
+}
```

Comment:
Acá hay que agregar `materiales Material[]` y `reparacionesNomencladas ReparacionNomenclada[]` para armar una relacion "many to many" https://playground.prisma.io/guides/relations_many-to-many-implicit?step=0

También `client_id` y `Usuario` tienen que ser `nullable`